rasa/train.py

```diff
@@ -293,16 +314,15 @@ def train_core(
 
     skill_imports = SkillSelector.load(config, stories)
 
-    if isinstance(domain, str):
-        try:
-            domain = Domain.load(domain, skill_imports)
-            domain.check_missing_templates()
-        except InvalidDomain as e:
-            print_error(
-                "Could not load domain due to: '{}'. To specify a valid domain path "
-                "use the '--domain' argument.".format(e)
-            )
-            return None
+    try:
+        domain = Domain.load(domain, skill_imports)
+        domain.check_missing_templates()
+    except InvalidDomain:
+        print_error(
+            "Core training is skipped because no domain was found. "
+            "Please specify a valid domain using '--domain' argument or check if provided domain file does exists"
```

Comment:
same small edits as the message above (obviously no nlu-specific message though)

Reply:
hi @erohmensing ,
it has been resolved